src/containerapp/azext_containerapp/commands.py

```diff
@@ -109,6 +109,9 @@ def load_command_table(self, _):
     with self.command_group('containerapp compose') as g:
         g.custom_command('create', 'create_containerapps_from_compose')
 
+    with self.command_group('containerapp env workload-profile') as g:
+        g.custom_command('set', 'set_workload_profile', deprecate_info=g.deprecate(hide=True))
```

Comment:
could you add redirect

Reply:
Can not add redirect, because `set `= `update` + `add`:
command set support to create a new workload profile or update an existing one.
But command `add `only support add a new one, if the added name has been used, throw validate error;
command `update `only support update an existing one, if not existing, throw error.